app/exposureHistory.ts

```diff
@@ -15,30 +15,33 @@ export interface NoKnown {
   date: Posix;
 }
 
-export type ExposureDatum = Possible | NoKnown;
+export interface NoData {
+  kind: 'NoData';
+  date: Posix;
+}
+
+export type ExposureDatum = Possible | NoKnown | NoData;
 
 export type ExposureInfo = Record<Posix, ExposureDatum>;
 
 export type ExposureHistory = ExposureDatum[];
 
-export const toExposureHistory: (
-  exposureInfo: ExposureInfo,
-  calendar: Posix[],
-) => ExposureHistory = (exposureInfo, calendar) => {
-  return calendar.map((date: Posix) => {
-    if (exposureInfo[date]) {
-      return exposureInfo[date];
-    } else {
-      return {
-        kind: 'NoKnown',
-        date,
-      };
-    }
-  });
+export interface ExposureCalendarOptions {
+  initDate: Posix;
```

Comment:
what do you think about `startDate`, or `todaysDate`, its not clear to me what `init` stands for here? is it when we start counting? or is it the lastDay, is it todays date? it would be nice have this clear in the name.